core/storage/base_model/gae_models.py

```diff
@@ -250,11 +295,35 @@ def _reconstitute_from_snapshot_id(self, snapshot_id):
 
     @classmethod
     def _get_snapshot_id(cls, instance_id, version_number):
+        """Gets snapshot id.
+
+        Args:
+            instance_id: int.
```

Comment:
Did you check the type? Don't guess -- use logging.info() or print. Ditto for version_number.